docs/pages/access-controls/guides/hardware-key-support.mdx

```diff
@@ -107,22 +108,22 @@ cluster auth preference has been updated
 
 Once hardware key support is enforced, affected users will be required to have their login sessions backed by a Hardware Key for all Teleport requests.
 
-These users will be prompted to connect and touch their YubiKey on log in:
+Affected users will be prompted to connect and touch their YubiKey on log in. If this is their first login, they may be required to immediately relogin.
 
 <ScopedBlock scope={["oss"]}>
```

Comment:
We only support Enterprise and Cloud so this can be removed.  It won't be visible

Reply:
Better to show it than have blanks in the page, this was an issue in the past - https://github.com/gravitational/teleport/issues/24211